manifests/base/config/metrics_allowlist.yaml

```diff
@@ -7,7 +7,7 @@ data:
     names:
       - :node_memory_MemAvailable_bytes:sum
       - ALERTS
-      - acm_policyreport_info
+      - policyreport_info
```

Comment:
@xrajesh this will need to change as well: https://github.com/open-cluster-management/multicluster-observability-operator/blob/main/manifests/base/alertmanager/alert_rules.yaml#L40

Reply:
after update the name, please inject the metrics name as alphabetical order even some of existing ones not follow that

Reply:
@bjoydeep @marcolan018 addressed both your comments . Please review.